be/src/vec/exec/join/vnested_loop_join_node.cpp

```diff
@@ -347,8 +347,8 @@ void VNestedLoopJoinNode::_process_left_child_block(MutableBlock& mutable_block,
 
 void VNestedLoopJoinNode::_update_additional_flags(Block* block) {
     if (_is_outer_join) {
-        auto p0 = _tuple_is_null_left_flag_column->assume_mutable();
```

Comment:
What is the difference between these two ways?

Reply:
for a column which would be reused, shouldn't use `assume_mutable()` to move its ownership. `mutate()` will check its' ownership by `ref_count` in it. then decide to use `assume_mutable()` or `clone()`. so it's always safe.
In this situation, `assume_mutable` lead to a low probability double release which clearly showed in our coredump file. so fixed it.